src/main/java/de/funfried/netbeans/plugins/external/formatter/java/google/GoogleFormatJob.java

```diff
@@ -68,10 +64,10 @@ public void format() throws BadLocationException {
 			if (setFormattedCode(code, formattedContent)) {
 				SwingUtilities.invokeLater(() -> {
 					if (pref.getBoolean(Settings.SHOW_NOTIFICATIONS, false)) {
-						NotificationDisplayer.getDefault().notify("Format using Goolge formatter", Icons.ICON_GOOGLE, "", null);
+						NotificationDisplayer.getDefault().notify("Format using Google formatter", Icons.ICON_GOOGLE, "", null);
```

Comment:
Thanks a lot for fixing this typo as well! But I'm afraid reordering the packages in existing classes is something which shouldn't be done by a contributor. Would be nice if you could revert that, but as I do not have any guidelines for raising a PR or for contribution in general, it's also ok if you leave it and I will do it then 🙂

Reply:
While I do not understand the reasoning on that one, I have complied and reverted it.

Reply:
The reason is simple, I've modified my NetBeans configuration, so it has the same package ordering as Eclipse, so I have the same results as my colleges, but that's not documented anywhere in the project, so you can't know. It's fine, I'll revert it.